configs/stdc/fcn_stdc-813_512x1024_40k_cityscapes.py

```diff
@@ -0,0 +1,4 @@
+_base_ = [
+    '../_base_/models/fcn_stdc-813.py', '../_base_/datasets/cityscapes.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_40k.py'
```

Comment:
The original repo takes 60k with customized pretrained model, thus we would use 80k or 160k schedule in the future.